src/UI/Art/SpriteArtist/CommanderOverlayArtist.java

```diff
@@ -15,9 +15,7 @@
   public static final int OVERLAY_WIDTH = 97;
   public static final int OVERLAY_HEIGHT = 20;
 
-  private static int animIndex = 0;
-  private static long animIndexUpdateTime = 0;
-  private static final int animIndexUpdateInterval = 8;
+  private static final int animIndexUpdateInterval = 13;
```

Comment:
Why change from 8 to 13?

Reply:
Because even at 13, it's faster than it was before. Not sure why.